Signal/Images.xcassets/empty-group-avatar.imageset/Contents.json

```diff
@@ -2,7 +2,16 @@
   "images" : [
     {
       "idiom" : "universal",
-      "filename" : "group-avi.pdf"
+      "filename" : "empty-group-avatar.png",
+      "scale" : "1x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "2x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "3x"
     }
   ],
   "info" : {
```

Comment:
We could introduce separate flavors of empty-group-avatar for the various contexts in which it is used.